python-checks/src/test/resources/checks/nonCallableCalled.py

```diff
@@ -255,3 +255,13 @@ def my_rec(x):
         my_rec(True)
     finally:
         my_rec = None
+
+def call_non_callable_property():
+    e = OSError()
+    e.errno()  # FN
+
+def member_access():
```

Comment:
I think you misread my comment. We already had that test showing a FN. I was asking for another test showing a new TP instead.

Reply:
removed